plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/quantity-selector/block.json

```
@@ -1,15 +1,8 @@
 {
 	"name": "woocommerce/add-to-cart-with-options-quantity-selector",
 	"title": "Quantity Selector (Experimental)",
-	"description": "Display an input field to select the number of products to add to cart.",
+	"description": "Display an input field with buttons to select the number of products to add to cart.",
```

Not a strong opinion but maybe we could call it stepper here as well? WDYT?

```suggestion
	"description": "Display an input field with stepper to select the number of products to add to cart.",
```